pymc/distributions/distribution.py

```diff
@@ -601,6 +656,20 @@ def update(self, node: Node):
         return updates
 
 
+def dist_moment(rv, *args):
```

Comment:
In case it was confusing, this is how you would register it in the base class. Then you don't need to do it everytime a `CustomDist` is called, only when the user wants to provide a non-default moment

```suggestion
@_moment.register(CustomSymbolicDistRV)
def dist_moment(rv, *args):
```